app/src/main/java/org/simple/clinic/scanid/ScanSimpleIdUpdate.kt

```diff
@@ -55,7 +55,11 @@ class ScanSimpleIdUpdate @Inject constructor(
 
   private fun patientFoundByIdentifierSearch(patients: List<Patient>, identifier: Identifier): ScanSimpleIdEffect {
     return if (patients.size > 1) {
-      OpenShortCodeSearch(BpPassport.shortCode(identifier))
+      try {
+        OpenShortCodeSearch(BpPassport.shortCode(identifier))
+      } catch (e: Exception) {
+        OpenShortCodeSearch(identifier.value)
```

Comment:
FYI in this scenario, we are opening instant search to search by ID and not shortcode search. I will be making that change as part of [this](https://app.clubhouse.io/simpledotorg/story/3218/show-patients-with-national-health-id-when-searching-with-from-scan-screen) story.